proposals/csharp-9.0/records.md

```diff
@@ -197,6 +197,118 @@ If the containing record is abstract, the synthesized clone method is also abstr
 If the "clone" method is not abstract, it returns the result of a call to a copy constructor. 
 
 
+### Printing members: PrintMembers and ToString
+
+If the record is derived from `object`, the record includes a synthesized method equivalent to a method declared as follows:
+```C#
+protected virtual void PrintMembers(System.StringBuilder builder, bool includeSeparator);
```

Comment:
should this method have a compiler-reserved (unspeakable) name?

Reply:
I think that would make sense. We don't expect anyone to call that method.

Reply:
`includeSeparator`?  taht seems really esoteric.  why do we need/want any sort of customization.  i dn't remember discussing that in the LDM at all for example.

Reply:
Not' sure if we doc this is in the spec or not but this method will need to remain forever in records for binary compat reasons. Once we ship it it's forever ;)

Reply:
Having a **bool** parameter to specify that a separator is required seems questionable to me - I've seen an awful number of similar situations where using a **bool** effectively boxed in future directions, causing issues for maintenance. They're also obtuse to readers - I still have to look up what `.ConfigureAwait(false)` means even though I've written it hundreds of times.

I'd suggest either 

(a) Follow Steve McConnell's advice that a boolean parameter actually means you've got two methods disguised as one - and therefore split them out
(b) Replace the boolean with an enum (e.g. `PrintMembers(builder, Separator.Comma)`) to both make the call site readable and to allow for other options in the future.

Reply:
Updated to use an unspeakable name